__tests__/cypress.config.js

```diff
@@ -0,0 +1,3 @@
+import configure from '@heartexlabs/ls-test';
```

Comment:
Is it a good name for this dir? `__tests__` is usually used for tests in a context of some logical domain and here we have full cypress setup. Maybe `cy`, `cypress` or `tests/cypress`?

Reply:
makes sense. moved to `tests/functional`